main.tf

```diff
@@ -184,3 +184,45 @@ data "cloudflare_zones" "domain" {
     status      = "active"
   }
 }
+
+
+/*
+ * Create user for sildisco:LogUser to save data to DynamoDB
+ */
+resource "aws_iam_user" "idp-hub" {
+  name = "idp-hub-${local.app_env}"
+}
+
+resource "aws_iam_access_key" "idp-hub-user" {
+  user = aws_iam_user.idp-hub.name
+}
+
+
+/*
+ *  This is needed when an idp metadata entry includes the sildisco:LogUser authproc
+ *  the dynamodb table name is included in the authproc's configuration
+ */
+resource "aws_iam_user_policy" "hub_loguser" {
+  name = "IDP-Hub-Dynamodb"
+  user = aws_iam_user.idp-hub.name
+
+  policy = <<EOF
```

Comment:
Optional: the aws provider [docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_user_policy) give an example using `jsonencode`. Might be easier to read.